### ISSUE TITLE
Narrow Corner or Turn

### ISSUE THREAD

**Issue by ishtiak111** [Mechyard/traffic-concepts-and-ideas#11]
Narrow corners or turns can significantly contribute to traffic jams in Dhaka city due to several factors:

1. Reduced Flow of Vehicles: Narrow corners limit the number of vehicles that can pass through simultaneously, creating bottlenecks. When vehicles slow down to navigate these turns, it reduces the overall speed of traffic, causing a ripple effect that can extend far back.

2. Increased Congestion: In densely populated areas like Dhaka, narrow roads and sharp turns often become choke points. Vehicles have to decelerate, stop, or maneuver carefully, leading to queues that build up quickly, especially during peak hours.

3. Frequent Accidents and Breakdowns: Narrow turns increase the likelihood of minor accidents or vehicle breakdowns, further blocking traffic and exacerbating congestion.

4. Impact on Public Transport: Buses and larger vehicles face more difficulty navigating narrow turns, often leading to longer dwell times at intersections and stops. This can cause delays not only for the buses but also for the vehicles behind them.

**Comment by ahmedsadatsakif** [comment]
Yes this is an excellent point. Using curved corners had become a norm in every developed country and we should work with the policy implementation part to incorporate this within the road construction guidelines if it's not there. 

Most of the corners are not suitable for turning cars or 4 wheel vehicles since they require a circular motion to turn.
